validator/engine/validator.js

```diff
@@ -5753,6 +5753,10 @@ amp.validator.ValidationResult.prototype.outputToTerminal = function(
   const {status} = this;
   if (status === amp.validator.ValidationResult.Status.PASS) {
     terminal.info('AMP validation successful.');
+    terminal.info('Important: Valid AMP pages should also be tested for proper '
```

Comment:
Let's avoid starting with `important` and find a shorter to the point message.

Reply:
@CrystalOnScript can we get an updated Troubleshooting page so that we can make this change of linking to that page, along with new text to go with it?

Reply:
@alabiaga @honeybadgerdontcare @Gregable - I forgot we also have [CORS in AMP - Testing CORS in AMP](https://www.ampproject.org/docs/fundamentals/amp-cors-requests#testing-cors-in-amp). Does this seem like a valid doc to link out to?

If not, I'm happy to set up a time to figure out exactly what should be added and where. Thanks!

Reply:
Suggestions:
"Succeed on the web by following our 'Publishing checklist' before distributing AMP documents"

"Successful AMP documents follow our 'Publishing checklist'."

"Review our 'Publishing checklist' to ensure successful AMP document distribution"

Reply:
Thanks, will update once the page you made propagate to the live site.